### ISSUE TITLE
expectedArguments string literals from meta appear twice

### ISSUE THREAD

**Issue by King2500** [klesun/deep-assoc-completion#78]
Sample code:
```php
class A
{
    public function getFoo($key) {
        return false;
    }
}

$a = new A();
$a->getFoo('<caret>');
```

`.phpstorm.meta.php`:
```php
namespace PHPSTORM_META {
    expectedArguments(\A::getFoo(), 0, 'test', 'val');
}
```

Literals appear twice:
![image](https://user-images.githubusercontent.com/4896363/57358589-0e35d780-7176-11e9-8594-7d24388fcc10.png)

Looks like your plugin is replicating standard PhpStorm expectedArguments behavior. Why?

**Comment by klesun** [comment]
Oh, my bad. Didn't check for built-in suggestions when adding my own.

**Comment by klesun** [comment]
Fixed in [2019.05.08.001](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/62063)

**Comment by King2500** [comment]
👍
